sdks/java/harness/src/test/java/org/apache/beam/fn/harness/FnApiDoFnRunnerTest.java

```diff
@@ -505,10 +516,226 @@ public void testSideInputIsAccessibleForDownstreamCallers() throws Exception {
     assertEquals(stateData, fakeClient.getData());
   }
 
+  private static class TestTimerfulDoFn extends DoFn<KV<String, String>, String> {
+    private static final TupleTag<String> mainOutput = new TupleTag<>("mainOutput");
```

Comment:
How and where is this used?

Reply:
Its not, I copied an example and didn't clean-up unused members.